ara/callback/log_ara.py

```diff
@@ -121,6 +139,10 @@ def log_task(self, result, status, **kwargs):
 
         db.session.add(self.taskresult)
 
+        if self.task.action == 'setup':
+            values = json.dumps(result._result['ansible_facts'])
+            self.create_or_update_facts(host, values)
```

Comment:
There's probably something missing for this specific bit to work. I haven't been able to (quickly) get it right without a callback error. Will look tomorrow.